src/sentry/models/organization.py

```diff
@@ -165,7 +165,7 @@ class Organization(Model, OrganizationAbsoluteUrlMixin, SnowflakeIdMixin):
 
     __include_in_export__ = True
     name = models.CharField(max_length=64)
-    slug = models.SlugField(unique=True)
+    slug: str = models.SlugField(unique=True)
```

Comment:
@markstory Just wanted to double check that this is sufficient to solve the type issues with slug for now 😬

Reply:
I think `slug: models.SlugField[str, str] = ...` should fix it too perhaps?

the django types for this are a little annoying but its `[GetType, SetType]`

Reply:
That does indeed work! Updated the def to use this instead.